seaworthy/containers/rabbitmq.py

```diff
@@ -0,0 +1,104 @@
+from seaworthy.logs import output_lines
+from .base import ContainerBase
+
+
+def _parse_rabbitmq_user(user_line):
+    user_tags = user_line.split('\t', 1)
+    if len(user_tags) != 2:  # pragma: no cover
+        raise RuntimeError()
```

Comment:
Maybe just delete this check completely? A `RuntimeError` without a message is possibly less use than the regular error we'd get. And this isn't supposed to be a very robust parser.

Reply:
Good point.